test/Vault.spec.ts

```diff
@@ -211,7 +213,20 @@ describe("Vault", () => {
           owner.address,
           PERFORMANCE_FEE_PCT
         )
-      ).to.be.revertedWith("Vault: minLockPeriod cannot be 0");
+      ).to.be.revertedWith("Vault: invalid minLockPeriod");
+    });
+
+    it("it reverts if min lock period is greater than maximum", async () => {
```

Comment:
duplicate "it"

Reply:
do you mean, we need to change "it reverts..." to "reverts..." ?

Reply:
Yap, like in the other tests

Reply:
fixed